tools/submission/submission_checker.py

```diff
@@ -277,70 +339,71 @@
             "dlrm-v2-99.9",
             "3d-unet-99",
             "3d-unet-99.9",
-            "gptj-99",
-            "gptj-99.9",
+            "llama3.1-8b",
             "llama2-70b-99",
             "llama2-70b-99.9",
-            "llama2-70b-interactive-99",
-            "llama2-70b-interactive-99.9",
             "stable-diffusion-xl",
             "mixtral-8x7b",
             "llama3.1-405b",
             "rgat",
             "pointpainting",
+            "deepseek-r1",
+            "whisper"
         ],
         "required-scenarios-datacenter": {
-            "resnet": ["Server", "Offline"],
             "retinanet": ["Server", "Offline"],
             "dlrm-v2-99": ["Server", "Offline"],
             "dlrm-v2-99.9": ["Server", "Offline"],
             "3d-unet-99": ["Offline"],
             "3d-unet-99.9": ["Offline"],
-            "gptj-99": ["Server", "Offline"],
-            "gptj-99.9": ["Server", "Offline"],
+            "llama3.1-8b": ["Server", "Offline"],
             "llama2-70b-99": ["Server", "Offline"],
             "llama2-70b-99.9": ["Server", "Offline"],
-            "llama2-70b-interactive-99": ["Server", "Offline"],
-            "llama2-70b-interactive-99.9": ["Server", "Offline"],
             "stable-diffusion-xl": ["Server", "Offline"],
             "mixtral-8x7b": ["Server", "Offline"],
             "llama3.1-405b": ["Server", "Offline"],
             "rgat": ["Offline"],
+            "deepseek-r1": ["Server", "Offline"],
+            "whisper": ["Offline"],
+        },
+        "optional-scenarios-datacenter": {
+            "llama2-70b-99": ["Interactive"],
```

Comment:
Are we making non-interactive Server scenario mandatory?

Reply:
My thinking was: if you run interactive, the same run is valid for the server scenario. So, the submitter could place it in both. But you are correct, it is better to avoid duplicated results. I'll do a patch for this

Reply:
This has been patched